plume-cli/src/users.rs

```diff
@@ -59,6 +64,7 @@ fn new<'a>(args: &ArgMatches<'a>, conn: &Connection) {
     let admin = args.is_present("admin");
     let bio = args.value_of("biography").unwrap_or("").to_string();
     let email = args.value_of("email").map(String::from).unwrap_or_else(|| super::ask_for("Email address"));
+    let custom_domain = args.value_of("custom-domain").unwrap_or("").to_string();
```

Comment:
I think you shouldn't unwrap. If it's None, then create the new user with custom_domain = None, otherwise a user with no custom domain would be stored as having the custom domain "" (empty string)

Reply:
@fdb-hiroshima do we just `.clone()` here?

Reply:
I think I would do
```rust
    let custom_domain = args.value_of("custom-domain").map(str::to_owned);
```

Reply:
thanks!